lua/wikis/clashroyale/MatchSummary.lua

```diff
@@ -126,12 +127,21 @@ function CustomMatchSummary._getSubMatchOpponentsAndPlayers(match, subMatch, sub
 	subMatch.opponents = Array.map(Array.range(1, #subMatch.players), function(opponentIndex)
 		local score, status = MatchGroupInputUtil.computeOpponentScore(
 			{opponentIndex = opponentIndex},
-			FnUtil.curry(MatchGroupInputUtil.computeMatchScoreFromMapWinners, subMatch.games)
+			FnUtil.curry(CustomMatchSummary.computeSubMatchScore, subMatch.games)
 		)
 		return {score = score, status = status}
 	end)
 end
 
+---@param games {winner: integer?, opponents: {score: integer?}[]}[]
+---@param opponentIndex integer
+---@return integer
+function CustomMatchSummary.computeSubMatchScore(games, opponentIndex)
+	return Array.reduce(Array.map(games, function(game)
+		return (game.opponents[opponentIndex] or {}).score or (game.winner == opponentIndex and 1 or 0)
+	end), Operator.add)
```

Comment:
This can return nil (if games list is empty). Consider adding a default `0` if `return integer` is correct